src/function.rs

```diff
@@ -48,26 +49,12 @@ impl<'instance> Function<'instance> {
     }
 
     #[allow(non_upper_case_globals)]
+    #[allow(non_snake_case)]
     fn parse_result(
         &self,
-        instance: &Instance<'instance>,
+        result_types: Vec<u8>,
```

Comment:
if following https://github.com/bytecodealliance/wamr-rust-sdk/pull/118/files#r1957496797, no need to pass `result_types`